crates/kumod/src/dmarc.rs

```diff
@@ -87,19 +87,44 @@ pub fn register<'lua>(lua: &'lua Lua) -> anyhow::Result<()> {
                 .check(&**resolver)
                 .await;
 
-                Ok(lua.to_value_with(
-                    &CheckHostOutput {
-                        disposition: result.result.clone(),
-                        result: AuthenticationResult {
-                            method: "dmarc".to_string(),
-                            method_version: None,
-                            result: result.result.to_string(),
-                            reason: Some(result.context),
-                            props: BTreeMap::default(),
-                        },
-                    },
-                    serialize_options(),
-                ))
+                match result.result {
+                    Disposition::Pass |
+                    Disposition::None |
+                    Disposition::TempError |
+                    Disposition::PermError => {
+                        Ok(lua.to_value_with(
+                            &CheckHostOutput {
+                                disposition: result.result.clone(),
+                                result: AuthenticationResult {
+                                    method: "dmarc".to_string(),
+                                    method_version: None,
+                                    result: result.result.to_string().to_ascii_lowercase(),
+                                    reason: Some(result.context),
+                                    props: BTreeMap::default(),
+                                },
+                            },
+                            serialize_options(),
+                        ))
+                    }
+                    disp @ Disposition::Quarantine |
+                    disp @ Disposition::Reject => {
+                        let mut props = BTreeMap::default();
+                        props.insert("policy", disp.to_string().to_ascii_lowercase());
```

Comment:
having looked through the RFC in more detail, the policy doesn't appear to be a thing that gets pulled through into `props` for dmarc; there's no defined property name for it.  I think that it is fine for it to be returned here via `CheckHostOutput::disposition` (although we should probably rename `CheckHostOutput` to something like `DmarcPolicyResult` or something, because `CheckHostOutput` is copypasta from the spf bindings!